lib/plugins/aws/provider/awsProvider.js

```diff
@@ -72,14 +78,18 @@ const impl = {
 
 class AwsProvider {
   static getProviderName() {
-    return 'aws';
+    return constants.providerName;
   }
 
-  constructor(serverless) {
+  constructor(serverless, options) {
+    this.naming = { sdk: this };
```

Comment:
this makes sure the naming module has access to the SDK as well, check the assign below